packages/hide-bazel-files/WORKSPACE

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+workspace(name = "npm_bazel_hide_bazel_files")
```

Comment:
making this a nested workspace seems unnecessarily complex. why not treat it like `@bazel/create`?

Reply:
e2e/symlinked_node_modules_npm depends on it and npm uses symlinks with `file:` deps so depending on `../../packages/hide-bazel-files` doesn't work there as can rename the `node_modules/@bazel/hide-bazel-files/BUILD.bazel` in there as its a symlink to `/packages/hide-bazel-files/BUILD.bazel`.